src/functionalTest/java/uk/gov/hmcts/reform/feature/helper/PasswordEncrypter.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.feature.helper;
+
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+public class PasswordEncrypter {
+
+    private PasswordEncrypter() {
+    }
+
+    public static String encryptUsingBCryptEncoder(String plaintextPassword) {
+        return new BCryptPasswordEncoder().encode(plaintextPassword);
```

Comment:
this is hashing, not encryption

Reply:
that class is not required.Will remove it